pym/bob/archive.py

```diff
@@ -991,9 +987,6 @@ def __init__(self, archiver, response):
     def __enter__(self):
         return (None, self.response)
     def __exit__(self, exc_type, exc_value, traceback):
-        # reset connection on abnormal termination
```

Comment:
I *think* we must call `self.response.close()` here unconditionally. The urllib library uses `Connection: close`. So if everything goes well, the socket will be closed by the server. But if we are interrupted in the middle of a response, the connection will hang. Even if everything is read, we should probably still close the connection from our side too.